apps/web/components/CheckoutPayment/CheckoutPayment.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div data-testid="checkout-payment" class="md:px-4 py-6">
+    <fieldset>
```

Comment:
Question: Is the `div` needed here? 
```suggestion
    <fieldset data-testid="checkout-payment" class="md:px-4 py-6">
```

Reply:
Otherwise styling is getting broken. `Legend` is positioning on the top of `fieldset` so instead of adding paddings to `fieldset` i had to wrap it with div that is adding paddings.
Without the div it looks like this:
![Zrzut ekranu 2023-07-28 o 09 15 49](https://github.com/vuestorefront/storefront-nuxt3-boilerplate/assets/44862757/ee8936b3-99ed-4d40-946b-b8cc27e6ac11)
but it should be like this:
![Zrzut ekranu 2023-07-28 o 09 16 28](https://github.com/vuestorefront/storefront-nuxt3-boilerplate/assets/44862757/a4973a2f-4058-4e7f-9d16-7216eb029860)

Reply:
I've did some research around that - let's keep the div wrapper. It seems that "fixing" fieldset & legend styling will just bring more confusion